resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -96,9 +96,26 @@ apprunner.view_service_log_streams.error=Unable to open log group {0}
 apprunner.view_service_log_streams.error_not_created=App Runner log group has not been created
 aws.description=Amazon Web Services (AWS) is a secure cloud services platform, offering compute power, database storage, content delivery and other functionality to help businesses scale and grow.
 aws.developer.tools.tab.title=Developer Tools
+aws.getstarted.auth.panel_bullet_iam=IAM Credentials
+aws.getstarted.auth.panel_bullets= IAM Identity Center\nAWS Builder ID
+aws.getstarted.resource.panel_title=Resource Explorer
+aws.getstarted.resource.panel_description=Work with S3, CloudWatch and more.
+aws.getstarted.resource.panel_question_text=Don't have an AWS account?
+aws.getstarted.auth.panel.notSupport_text=Not supported for this feature
 aws.notification.credentials_missing=Error: AWS credentials not configured
 aws.notification.sam_cli_not_valid=Error: {0}. Click <a href="">here</a> to reconfigure AWS SAM.
 aws.notification.title=AWS Toolkit
+aws.onboarding.getstarted.panel.bottom_text_question=Why do these have different authentication requirements?
+aws.onboarding.getstarted.panel.bottom_text=1 or 2 sentence about why these are diferent and a <a href="https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/auth-access.html">link to a doc and learn more</a> 
+aws.onboarding.getstarted.panel.button_iam_login=Authenticate with IAM
+aws.onboarding.getstarted.panel.comment_link_doc=<a href = "https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/welcome.html">Documentation</a> | 
+aws.onboarding.getstarted.panel.comment_link_github=<a href="https://github.com/aws/aws-toolkit-jetbrains">Join us on github</a> |
+aws.onboarding.getstarted.panel.group_title=<html><h2 style="color:{0}"; "font-size= 14px"><b>Select a feature to setup authentication for</b></h2></html>
```

Comment:
font/color style should be applied to the label directly instead of html

Reply:
sure